src/StoreApi/docs/cart-items.md

```diff
@@ -25,162 +25,164 @@ curl "https://example-store.com/wp-json/wc/store/v1/cart/items"
-		"id": 34,
-		"quantity": 1,
-		"quantity_limits": {
-			"minimum": 1,
-			"maximum": 99,
-			"multiple_of": 1,
-			"editable": true
-		},
-		"name": "WordPress Pennant",
-		"summary": "<p>This is an external product.</p>",
-		"short_description": "<p>This is an external product.</p>",
-		"description": "<p>Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Mauris placerat eleifend leo.</p>",
-		"sku": "wp-pennant",
-		"low_stock_remaining": null,
-		"backorders_allowed": false,
-		"show_backorder_badge": false,
-		"sold_individually": false,
-		"permalink": "https://local.wordpress.test/product/wordpress-pennant/",
-		"images": [
-			{
-				"id": 57,
-				"src": "https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1.jpg",
-				"thumbnail": "https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1-324x324.jpg",
-				"srcset": "https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1.jpg 800w, https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1-324x324.jpg 324w, https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1-100x100.jpg 100w, https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1-416x416.jpg 416w, https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1-300x300.jpg 300w, https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1-150x150.jpg 150w, https://local.wordpress.test/wp-content/uploads/2020/03/pennant-1-768x768.jpg 768w",
-				"sizes": "(max-width: 800px) 100vw, 800px",
-				"name": "pennant-1.jpg",
-				"alt": ""
-			}
-		],
-		"variation": [],
-		"item_data": [],
-		"prices": {
-			"currency_code": "GBP",
-			"currency_symbol": "£",
-			"currency_minor_unit": 2,
-			"currency_decimal_separator": ".",
-			"currency_thousand_separator": ",",
-			"currency_prefix": "£",
-			"currency_suffix": "",
-			"price": "1105",
-			"regular_price": "1105",
-			"sale_price": "1105",
-			"price_range": null,
-			"raw_prices": {
-				"precision": 6,
-				"price": "11050000",
-				"regular_price": "11050000",
-				"sale_price": "11050000"
-			}
-		},
-		"totals": {
-			"currency_code": "GBP",
-			"currency_symbol": "£",
-			"currency_minor_unit": 2,
-			"currency_decimal_separator": ".",
-			"currency_thousand_separator": ",",
-			"currency_prefix": "£",
-			"currency_suffix": "",
-			"line_subtotal": "1105",
-			"line_subtotal_tax": "0",
-			"line_total": "884",
-			"line_total_tax": "0"
-		},
-		"_links": {
-			"self": [
-				{
-					"href": "https://local.wordpress.test/wp-json/wc/store/v1/cart/items/e369853df766fa44e1ed0ff613f563bd"
-				}
-			],
-			"collection": [
-				{
-					"href": "https://local.wordpress.test/wp-json/wc/store/v1/cart/items"
-				}
-			]
-		}
-	}
+  {
+    "key": "9bf31c7ff062936a96d3c8bd1f8f2ff3",
+    "id": 15,
+    "type": "simple",
```

Comment:
This and line 109 are the only lines that changed in this file, not sure why the diff is showing the entire file.